### PR TITLE
Update resources.yml

### PR DESCRIPTION
**Pull request by httpJunkie**
Add recent resources for accessibility not yet covered in a11y project

### CODE REVIEW

File: _data/resources.yml [hunk 1]
```diff
@@ -146,6 +146,9 @@
     - url: https://pressbooks.library.ryerson.ca/wafd/
       title: "Web Accessibility for Developers – Open Textbook"
       description: "by The Chang School, Ryerson University"
+    - url: https://silktide.com/resources/book
```
Comment (ericwbailey) [inline _data/resources.yml]: Let's remove this for now. I don't feel comfortable promoting a book that isn't published yet without vetting its quality, especially if it's coming from a source that isn't primarily accessibility-focused.

File: _data/resources.yml [hunk 2]
```diff
@@ -383,6 +392,9 @@
     - url: https://www.youtube.com/watch?v=hUCOA338jwk
       title: "JavaScript and Civil Rights (Marcy Sutton)"
       description: "Web Rebels (2017)"
+    - url: https://www.youtube.com/watch?v=guXNEcwg6os
+      title: "Designing for Accessibility (Google I/O'19)"
+      description: "hear from people who are working to make the world an easier place for those with disabilities."
```
Comment (ericwbailey) [inline _data/resources.yml]: Capitalize the "h" in "hear".

File: _data/resources.yml (hunk 1 above)
Comment (httpJunkie) [reply _data/resources.yml]: No problem. I'm waiting on that one for release maybe we can add it later.

**Comment by httpJunkie** [comment]
I have made these changes. Thanks for the heads up!

**Comment by ericwbailey** [comment]
Awesome, thank you for your contribution!

**Pull request merged**